### ISSUE TITLE
I got the same error. What do i do?

### ISSUE THREAD

**Issue by iwantcustom** [TheOfficialFloW/VitaShell#593]
I recently bought a vita to hack and after 3 days I finally got to set up h-encore. But when i try to download vitashell it gives me the error code 80431075 and it fails. I've tried personal hotspot and restarting wifi but it didn't work. What do i do now?

**Comment by TheOfficialFloW** [comment]
Will fix it this week. If not please ping here again.

**Comment by TheOfficialFloW** [comment]
Follow https://github.com/TheOfficialFloW/VitaShell/issues/458#issuecomment-761523772

**Comment by mannyzv** [comment]
> Follow [#458 (comment)](https://github.com/TheOfficialFloW/VitaShell/issues/458#issuecomment-761523772)

yes i do but it installs it as a 5kb file on the vita nd errors when u copy
